pom.xml

```diff
@@ -1479,16 +1479,6 @@
 
   <repositories>
     <repository>
-      <id>jboss-snapshots</id>
-      <url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-    </repository>
-    <repository>
```

Comment:
Why removing the JBoss snapshots repo and keep Sonatype snapshots ?
PNC shapshots are published at JBoss one.